code/src/index.css

```diff
@@ -11,3 +11,115 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+.questions {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: black;
+  height: 100vh;
+  width: 100vw;
+  background-image: url(./components/images/start-img.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+h2 {
+  color: white;
+  font-size: 35px;
+}
+#nameInput {
+  height: 50px;
+  width: 12rem;
+  margin: 10px;
+  text-align: center;
+  font-size: large;
+  font-weight: 500;
+  background: rgba(32, 31, 31, 0.863);
+  color: white;
+  border-radius: 10px;
+  border: 3px solid white;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  z-index: 1;
+  color: white;
+  text-align: justify;
+}
+.first-que {
+  margin: 20px;
+  font-weight: 500;
+  font-size: 20px;
+  letter-spacing: 1px;
+  text-align: justify;
+}
+.container {
+  width: 500px;
+  background: rgba(32, 31, 31, 0.5);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
Very good that you re-use divs for different question to make the code smooth and short.